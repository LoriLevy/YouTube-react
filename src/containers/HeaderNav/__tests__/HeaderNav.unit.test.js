import React from "react";
import { shallow } from "enzyme";
import HeaderNav from "../HeaderNav";

describe("HeaderNav", () => {
  test("it renders", () => {
    const wrapper = shallow(<HeaderNav />);
    expect(wrapper).toMatchSnapshot();
  });
});
