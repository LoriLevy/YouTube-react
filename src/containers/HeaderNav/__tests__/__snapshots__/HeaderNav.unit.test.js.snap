// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderNav It renders the HeaderNav 1`] = `
<Menu
  borderless={true}
  className="top-menu"
  fixed="top"
>
  <MenuItem
    className="logo"
    header={true}
  >
    <Image
      as="img"
      size="tiny"
      src="logo.jpg"
      ui={true}
    />
  </MenuItem>
  <MenuMenu
    className="nav-container"
  >
    <MenuItem
      className="search-input"
    >
      <Form
        as="form"
      >
        <FormField>
          <Input
            action="Go"
            placeholder="Search"
            size="small"
            type="text"
          />
        </FormField>
      </Form>
    </MenuItem>
    <MenuMenu
      position="right"
    >
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="video camera"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="grid layout"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="chat"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="alarm"
          size="large"
        />
      </MenuItem>
      <MenuItem
        name="avatar"
      >
        <Image
          as="img"
          avatar={true}
          src="http://via.placeholder.com/80x80"
          ui={true}
        />
      </MenuItem>
    </MenuMenu>
  </MenuMenu>
</Menu>
`;

exports[`HeaderNav iIt renders the HeaderNav 1`] = `
<Menu
  borderless={true}
  className="top-menu"
  fixed="top"
>
  <MenuItem
    className="logo"
    header={true}
  >
    <Image
      as="img"
      size="tiny"
      src="logo.jpg"
      ui={true}
    />
  </MenuItem>
  <MenuMenu
    className="nav-container"
  >
    <MenuItem
      className="search-input"
    >
      <Form
        as="form"
      >
        <FormField>
          <Input
            action="Go"
            placeholder="Search"
            size="small"
            type="text"
          />
        </FormField>
      </Form>
    </MenuItem>
    <MenuMenu
      position="right"
    >
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="video camera"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="grid layout"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="chat"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="alarm"
          size="large"
        />
      </MenuItem>
      <MenuItem
        name="avatar"
      >
        <Image
          as="img"
          avatar={true}
          src="http://via.placeholder.com/80x80"
          ui={true}
        />
      </MenuItem>
    </MenuMenu>
  </MenuMenu>
</Menu>
`;

exports[`HeaderNav it renders 1`] = `
<Menu
  borderless={true}
  className="top-menu"
  fixed="top"
>
  <MenuItem
    className="logo"
    header={true}
  >
    <Image
      as="img"
      size="tiny"
      src="logo.jpg"
      ui={true}
    />
  </MenuItem>
  <MenuMenu
    className="nav-container"
  >
    <MenuItem
      className="search-input"
    >
      <Form
        as="form"
      >
        <FormField>
          <Input
            action="Go"
            placeholder="Search"
            size="small"
            type="text"
          />
        </FormField>
      </Form>
    </MenuItem>
    <MenuMenu
      position="right"
    >
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="video camera"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="grid layout"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="chat"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="alarm"
          size="large"
        />
      </MenuItem>
      <MenuItem
        name="avatar"
      >
        <Image
          as="img"
          avatar={true}
          src="http://via.placeholder.com/80x80"
          ui={true}
        />
      </MenuItem>
    </MenuMenu>
  </MenuMenu>
</Menu>
`;

exports[`It renders the HeaderNav 1`] = `
<Menu
  borderless={true}
  className="top-menu"
  fixed="top"
>
  <MenuItem
    className="logo"
    header={true}
  >
    <Image
      as="img"
      size="tiny"
      src="logo.jpg"
      ui={true}
    />
  </MenuItem>
  <MenuMenu
    className="nav-container"
  >
    <MenuItem
      className="search-input"
    >
      <Form
        as="form"
      >
        <FormField>
          <Input
            action="Go"
            placeholder="Search"
            size="small"
            type="text"
          />
        </FormField>
      </Form>
    </MenuItem>
    <MenuMenu
      position="right"
    >
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="video camera"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="grid layout"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="chat"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="alarm"
          size="large"
        />
      </MenuItem>
      <MenuItem
        name="avatar"
      >
        <Image
          as="img"
          avatar={true}
          src="http://via.placeholder.com/80x80"
          ui={true}
        />
      </MenuItem>
    </MenuMenu>
  </MenuMenu>
</Menu>
`;

exports[`Itrenders HeaderNav 1`] = `
<Menu
  borderless={true}
  className="top-menu"
  fixed="top"
>
  <MenuItem
    className="logo"
    header={true}
  >
    <Image
      as="img"
      size="tiny"
      src="logo.jpg"
      ui={true}
    />
  </MenuItem>
  <MenuMenu
    className="nav-container"
  >
    <MenuItem
      className="search-input"
    >
      <Form
        as="form"
      >
        <FormField>
          <Input
            action="Go"
            placeholder="Search"
            size="small"
            type="text"
          />
        </FormField>
      </Form>
    </MenuItem>
    <MenuMenu
      position="right"
    >
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="video camera"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="grid layout"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="chat"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="alarm"
          size="large"
        />
      </MenuItem>
      <MenuItem
        name="avatar"
      >
        <Image
          as="img"
          avatar={true}
          src="http://via.placeholder.com/80x80"
          ui={true}
        />
      </MenuItem>
    </MenuMenu>
  </MenuMenu>
</Menu>
`;

exports[`renders HeaderNav 1`] = `
<Menu
  borderless={true}
  className="top-menu"
  fixed="top"
>
  <MenuItem
    className="logo"
    header={true}
  >
    <Image
      as="img"
      size="tiny"
      src="logo.jpg"
      ui={true}
    />
  </MenuItem>
  <MenuMenu
    className="nav-container"
  >
    <MenuItem
      className="search-input"
    >
      <Form
        as="form"
      >
        <FormField>
          <Input
            action="Go"
            placeholder="Search"
            size="small"
            type="text"
          />
        </FormField>
      </Form>
    </MenuItem>
    <MenuMenu
      position="right"
    >
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="video camera"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="grid layout"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="chat"
          size="large"
        />
      </MenuItem>
      <MenuItem>
        <Icon
          as="i"
          className="header-icon"
          name="alarm"
          size="large"
        />
      </MenuItem>
      <MenuItem
        name="avatar"
      >
        <Image
          as="img"
          avatar={true}
          src="http://via.placeholder.com/80x80"
          ui={true}
        />
      </MenuItem>
    </MenuMenu>
  </MenuMenu>
</Menu>
`;
